script.js

```diff
@@ -1,29 +1,37 @@
 //You can edit ALL of the code here
+
+////////////  **********               Episodes           **********////////////
+// This variable contain all Episodes
 let allEpisodes;
```

Comment:
hello, Ahmad you have done excellent job very nice and clean code easy to read much better than my code. only think I am not sure on line 5 let allEpisodes wouldn't be a better to put that inside function then return it  where you want to use maybe I am wrong but I here that leaving variable on global scope should be avoided. but I really like you coding style.

Reply:
Hi Ali,

Thanks for your feedback, I put allEpisode as a global variable because I want to call the API only once and I wanna use it many times.

Regards,